fares-data-build-tool/create-data
# fix: Fixed indexing issue for passenger types

# Description

-   Allowing multiple of the same passenger type to be in the same group.

# Testing instructions

-   Use test matching JSON with 2 adults in the same group, see that it creates valid netex.

# Type of change
